lib/control.js

```diff
@@ -98,8 +98,23 @@ const sequential = (fns, context, callback) => {
   next();
 };
 
+const runIf = (condition, defaultVal, asyncFn, ...args) => {
```

Comment:
Just a nit: I think we should also add a documentation comment above the function.